webpack.config.dev.js

```diff
@@ -86,7 +87,8 @@ module.exports = (env = {}) => {
         })
     });
 
-    COMPONENTS_DEV_CONFIG.entry = ['US', 'US-EZP', 'DE', 'GB', 'FR'].reduce(
+    localeOptions.push('US-EZP');
```

Comment:
Can we do something like `[...localeOptions, 'US-EZP']` to avoid mutating the import?